scss/6-components/_oisc-warning.scss

```diff
@@ -0,0 +1,65 @@
+/* =========================================================================
+   The Office of the Immigration Services Commissioner (OISC)  Warnings
+   ========================================================================= */
+
+.cads-oisc-warning {
+  background-color: $cads-palette__light-grey;
+  border: 1px solid $cads-palette__dark-grey;
+  margin-bottom: $cads-spacing-3;
+
+  .cads-oisc-warning__title {
```

Comment:
does this need to be nested?  this will generate:
`.cads-oisc-warning .cads-oisc-warning__title`

If not wouldn't just `.cads-oisc-warning__title` be better? 

so either un-nested or use the union operator:
```
.cads-oisc-warning {
   ...

   &__title {
      ...
   }
}

Reply:
Personally, I think the nesting can be a useful defensive technique. i.e. by nesting it's not possible to accidentally use `cads-oisc-warning__title` without the parent `cads-oisc-warning` class for example. Not such a problem here I guess when the markup is mostly included from a partial, but a useful pattern nonetheless.

If the nesting is undesirable, I'd opt to just not nest things, e.g.

```scss
.cads-oisc-warning {
    margin: 2ch;    
}
.cads-oisc-warning__title {
    color: palegoldenrod;
}
```

Reply:
Agreed to avoid keep specificity as low as possible unless we find a reason not to.  Will keep a track of instances where BEM element selectors are used outside of their block.